src/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -0,0 +1,76 @@
+
+Client diversity also offers resilience to attacks. For example, an attack that [tricks a particular client](https://twitter.com/vdWijden/status/1437712249926393858) onto a particular branch of the chain is unlikely to be successful because other clients are unlikely to be exploitable in the same way and the canonical chain remains uncorrupted. Low client diversity increases the risk associated with a hack on the dominant client.
+
+After the merge to proof-of-stake some of these risks become more acute because a critical bug in a consensus client with >33% share of all nodes could prevent the Beacon Chain from finalizing, stalling Ethereum. A client with both a 2/3 majority and a bug affecting consensus could cause the chain to [incorrectly split and then finalize](https://www.symphonious.net/2021/09/23/what-happens-if-beacon-chain-consensus-fails/), leading to a large set of validators being stuck on an incorrect chain. These validators face slashing or a slow and expensive voluntary withdrawal and reactivation if they want to rejoin the correct chain. The magnitude of a slashing scales with the number of culpable nodes with a 2/3 majority slashed maximally (32 ETH).
+
+Although these are unlikely scenarios, the risk associated with them can be mitigated by evening out the distribution of clients across the active nodes. Ideally, no consensus client would ever have more than 33% share of the total nodes.
+
+At the same time, there is a human cost to single client dominance because it puts excess strain and responsibility on one small team of developers. The smaller the client diversity, the greater the consequences of a mistake for the developers maintaining the dominant client. Spreading this load across multiple teams is good for the health of Ethereum's network of nodes and network of people.
+
+## Use a minority client {use-minority-client}
+
+Addressing client diversity requires more than individual users to choose minority clients - it requires mining/validator pools and institutions like the major dapps and exchanges to switch clients too. However, all users can do their part in redressing the current imbalance and normalising the use of all the available Ethereum software. After the merge, all node operators will be required to run an execution client and a consensus client. Choosing combinations of the clients suggested below will help increase client diversity.
+
+### Execution Clients
+
+[Besu](https://www.hyperledger.org/use/besu)
+
+[Nethermind](https://downloads.nethermind.io/)
+
+[Erigon](https://github.com/ledgerwatch/erigon)
+
+[CoreGeth](https://core-geth.org/)
+
+### Consensus Clients
+
+[Nimbus](https://nimbus.team/)
+
+[Lighthouse](https://github.com/sigp/lighthouse)
+
+[Teku](https://consensys.net/knowledge-base/ethereum-2/teku/)
+
+and soon [Lodestar](https://github.com/ChainSafe/lodestar)
```

Comment:
Where are we at with this?

Reply:
afaik lodestar has been running very nicely for quite a while now but is not being promoted as production-grade yet until it's been battle tested a bit more. But someone more clued-in can hopefully advise.

Reply:
This is my understanding as well... Haven't heard any further updates on this recently

Reply:
Have asked about this on Eth R&D discord and will await advice!

Reply:
Heard back in Discord... gonna lean towards labeling Lodestar as under review and audit.

```suggestion
[Lodestar](https://github.com/ChainSafe/lodestar) - _Under review and audit_
```

This was also noted in #5242 ([see comment](https://github.com/ethereum/ethereum-org-website/pull/5242#discussion_r794154382))